core: add more commands to the plugin (#18)
rook/kubectl-rook-ceph: README.md

### Commands

- `ceph <args>` : Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.

- `rbd <args>` : Call a 'rbd' CLI command with arbitrary args"

- `mon-endpoints` : print mon endpoints"

> - `operator <subcommands>` : run sub-commands against the Rook-Ceph operator mentioned below

Reviewer: I think we don't need to mention the details, it seems obvious the subcommands are below, especially when they look at the later examples.
```suggestion
- `operator`
```

  - `restart` : restart the Rook-Ceph operator
  - `<property> <value>` : Set the property in the rook-ceph-operator-config configmap.

> - `rook <subcommands>` : run sub-commands mentioned below

Reviewer: ```suggestion
- `rook`
```

>   - `version` : print the version of Rook and the Ceph daemons version from the CephCluster CR status"

Reviewer: No longer prints the cluster CR status, right?
```suggestion
  - `version` : print the version of Rook
```

>   - `status` : print the phase and/or conditions of every CR in the namespace"

Reviewer: Printing the status of all CRs seems verbose. What about these options?
```suggestion
  - `status` : print the phase and conditions of the CephCluster CR
  - `status all` : print the phase and conditions of all CRs 
  - `status <CR>` : print the phase and conditions of CRs of a specific type, such as `cephobjectstore`, `cephfilesystem`, etc 
```

Author: added.

- `help` : Output help text.

> ## Examples

Reviewer: How about adding an example for printing versions? It's one of the simplest and most common questions.
```
### Versions

kubectl rook-ceph rook version

<output>

kubectl rook-ceph ceph versions

<output>

### Commands

- `ceph <args>` : Run a Ceph CLI command. Supports any arguments the `ceph` command supports. See [Ceph docs](https://docs.ceph.com/en/pacific/start/intro/) for more.

- `rbd <args>` : Call a 'rbd' CLI command with arbitrary args"

- `mon-endpoints` : print mon endpoints"

- `operator`
  - `restart` : restart the Rook-Ceph operator

>   - `<property> <value>` : Set the property in the rook-ceph-operator-config configmap.

Reviewer: ```suggestion
  - `set <property> <value>` : Set the property in the rook-ceph-operator-config configmap.
```

>deployment.apps/rook-ceph-operator restarted
>```

> ### Output Rook version

Reviewer: ```suggestion
### Rook Version
```

```console
kubectl rook-ceph rook version
```

```text
rook: v1.8.0-alpha.0.267.g096dabfa6
go: go1.16.13
```

> ### Output all Ceph daemons version

Reviewer: ```suggestion
### Ceph Version
```
